cmd/global-heal.go

```diff
@@ -277,10 +324,10 @@ func (er *erasureObjects) healErasureSet(ctx context.Context, buckets []string,
 					versionID: "",
 				}, madmin.HealItemObject)
 				if err != nil {
-					tracker.ItemsFailed++
+					entryCh <- healEntryFailure(0)
```

Comment:
compete this with `context` @krisis  if context got canceled we don't have to block here infinitely.